### PR TITLE
Remove challengutils dependency

### PR DESCRIPTION
**Pull request by thomasyu888**
#3

### CODE REVIEW

File: synapseformation/create.py [hunk 1]
```diff
@@ -205,3 +203,59 @@ def get_or_create_queue(self, name: str, *args,
...
+        """Gets the Challenge associated with a Project.
+
+        See the definition of a Challenge object here:
+        https://docs.synapse.org/rest/org/sagebionetworks/repo/model/Challenge.html
+
+        Args:
+            projectId: A Synapse Id of a Project.
+
+        Returns:
+            Challenge object
```
Comment (kdaily) [inline synapseformation/create.py]: dictionary in format of Synapse challenge object response

File: synapseformation/create.py [hunk 2]
```diff
@@ -205,3 +203,59 @@ def get_or_create_queue(self, name: str, *args,
         self.logger.info('{} Queue {}({})'.format(self._update_str,
                                                   queue.name, queue.id))
         return queue
+
+    def _get_challenge(self, projectId) -> 'Challenge':
```
Comment (kdaily) [inline synapseformation/create.py]: returns dict

File: synapseformation/create.py [hunk 3]
```diff
@@ -205,3 +203,59 @@ def get_or_create_queue(self, name: str, *args,
...
+            projectId: A Synapse Id of a Project.
+
+        Returns:
+            Challenge object
+        """
+        challenge = self.syn.restGET(f"/entity/{projectId}/challenge")
+        return challenge
+
+    def _create_challenge(self, projectId: str,
+                          participantTeamId: str) -> 'Challenge':
```
Comment (kdaily) [inline synapseformation/create.py]: return dict

File: synapseformation/create.py [hunk 4]
```diff
@@ -205,3 +203,59 @@ def get_or_create_queue(self, name: str, *args,
...
+
+        See the definition of a Challenge object here:
+        https://docs.synapse.org/rest/org/sagebionetworks/repo/model/Challenge.html
+
+        Args:
+            participantTeamId: An Entity or Synapse ID of a Project.
+            projectId: A Team or Team ID.
+
+        Returns:
+            Challenge object
```
Comment (kdaily) [inline synapseformation/create.py]: dict

File: synapseformation/create.py [hunk 5]
```diff
@@ -205,3 +203,59 @@ def get_or_create_queue(self, name: str, *args,
...
+        """
+        challenge_object = {'participantTeamId': participantTeamId,
+                            'projectId': projectId}
+        challenge = self.syn.restPOST('/challenge',
+                                      json.dumps(challenge_object))
+        return challenge
+
+    def get_or_create_challenge(self, projectId, *args, **kwargs) -> 'Challenge':
```
Comment (kdaily) [inline synapseformation/create.py]: dict

Comment (kdaily) [inline synapseformation/create.py]: only kwargs

**Review by kdaily — CHANGES_REQUESTED** [review]
LG overall, few changes! THanks

File: requirements.txt [hunk 6]
```diff
@@ -1,3 +1,2 @@
 click
-synapseclient==1.9.4
-git+https://github.com/Sage-Bionetworks/challengeutils.git@v1.3.0#egg=challengeutils
\ No newline at end of file
+synapseclient==1.9.4
```
Comment (thomasyu888) [inline requirements.txt]:
```suggestion
synapseclient==2.0.0
```

**Review by kdaily — APPROVED** [review]

**Pull request merged**